pkg/server/handler.go

```diff
@@ -47,6 +48,8 @@ func (ctrl *Controller) indexHandler() http.HandlerFunc {
 		} else if path == "/service-discovery" {
 			ctrl.statsInc("service-discovery")
 			ctrl.renderIndexPage(rw, r)
+		} else if path == "/login" || path == "/signup" {
```

Comment:
let's add other pages as well, particularly `/forbidden`, `/welcome` and `/redirect`

Reply:
Still discussing on the way it should be handled. Seems `/welcome` and `/redirect` does pretty much the same.